### ISSUE TITLE
invalid incoming T: invalid type: string "XXXXXXXX.XX", expected i64

### ISSUE THREAD

**Issue by tarikeshaq** [mozilla/application-services#5292]
We're seeing a few of those on sentry ref: https://sentry.io/organizations/mozilla/issues/3796658061/?project=6295546&query=is%3Aunresolved+rusterror&referrer=issue-stream&statsPeriod=14d

This is probably related to the fix for #5284 as it started showing up in AS 96.1.3

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-3520)

**Comment by mhammond** [comment]
> This is probably related to the fix for #5284 as it started showing up in AS 96.1.3

I can see some in 96.1.2 too. So to my mind, this makes it likely to be #5139 and might be unrelated to tabs.

Unfortunately, we don't seem to have breadcrumbs about what engine it was, but my money is on history. Let's assume that's true, then before #5284, I think this error would be handled [here](https://github.com/mozilla/application-services/blob/b813f022c40428c23ea1e523563896eb9d62cc76/components/places/src/history_sync/plan.rs#L191). In current main it comes from [here](https://github.com/mozilla/application-services/blob/a4dfd1f4d3e370df62e779670f04cad10fbbdc0f/components/places/src/history_sync/plan.rs#L187), via [this code](https://github.com/mozilla/application-services/blob/a4dfd1f4d3e370df62e779670f04cad10fbbdc0f/components/sync15/src/bso/content.rs#L176) - which uses the error reporter.

In other words, there seems a good chance that this problem always existed, but previously just did an unnoticed `log::warn!`. However, now it reports the error, so we are seeing it. ie, this might not actually be a regression.

**Comment by bendk** [comment]
Is this still happening?  I think we might be able to close it.
